doc/src/dump_modify.txt

```diff
@@ -620,6 +621,21 @@ threshold criterion is met.  Otherwise it is not met.
 
 :line
 
+The {units} keyword only applies to the dump {atom}, {custom}, and
```

Comment:
Clarify if these 2 lines are added at the beginning of every file dump
creates?  E.g. one file per snapshot, N files from P procs (once or per snapshot).

Reply:
yes, each file. i've added the word "individual" to this phrase, so it is more obvious.